docs/howto/export.md

```diff
@@ -119,11 +119,11 @@ Some files should be exported before others, e.g., a Spark `_SUCCESS` file expor
 the same prefix might send the wrong indication.
 
 The export operation may contain several *rounds* within the same export.
-A failing round will stop the export of all the files of the next `rounds`.
+A failing round will stop the export of all the files of the next rounds.
```

Comment:
what is a `round`? It's italicised at first and then not after. Should it be defined if it's a 'thing'?